src/ode_solver/rrk_explicit_ode_solver.h

```diff
@@ -0,0 +1,59 @@
+#ifndef __RRK_EXPLICIT_ODESOLVER__
+#define __RRK_EXPLICIT_ODESOLVER__
+
+#include "dg/dg.h"
+#include "ode_solver_base.h"
+#include "explicit_ode_solver.h"
+
+namespace PHiLiP {
+namespace ODE {
+
+/// Relaxation Runge-Kutta ODE solver derived from ExplicitODESolver.
+#if PHILIP_DIM==1
+template <int dim, typename real, typename MeshType = dealii::Triangulation<dim>>
+#else
+template <int dim, typename real, typename MeshType = dealii::parallel::distributed::Triangulation<dim>>
+#endif
+class RRKExplicitODESolver: public ExplicitODESolver <dim, real, MeshType>
+{
+public:
+    /// Default constructor that will set the constants.
+    RRKExplicitODESolver(std::shared_ptr< DGBase<dim, real, MeshType> > dg_input); ///< Constructor.
+
+    /// Destructor
+    ~RRKExplicitODESolver() {};
+
+    /// Function to evaluate the solution update 
+    /* Same as ExplicitODESolver except modifying timestep size
+     * No implementation for pseudotime
+     * Currently only explicit formulation for relaxation parameter is implemented 
+     * (i.e. Ketcheson 2019 method but not yet Ranocha 2020)
+     */
+    void step_in_time(real dt, const bool pseudotime) override;
+
+    /// Relaxation Runge-Kutta parameter gamma^n
+    /* See:  Ketcheson 2019, "Relaxation Runge--Kutta methods: Conservation and stability for inner-product norms"
+     *       Ranocha 2020, "Relaxation Runge--Kutta Methods: Fully Discrete Explicit Entropy-Stable Schemes for the Compressible Euler and Navier--Stokes Equations"
```

Comment:
is the plan to eventually do something similar to this Ranocha paper?

Reply:
Yes, hopefully! The method of finding the relaxation parameter I've implemented only works when the conserved/dissipated variable is energy. If it's entropy or some other convex functional, then the relaxation parameter needs to be found implicitly, as proposed in the Ranocha paper.

Reply:
Ok very cool